src/hooks/useAssetsBalanceForAddress.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
+import { Address } from 'viem';
+
+import { selectUserAssetsBalance, selectorFilterByUserChains } from '@/__swaps__/screens/Swap/resources/_selectors/assets';
+import { useUserAssets } from '@/__swaps__/screens/Swap/resources/assets';
+import { EthereumAddress, ParsedAddressAsset } from '@/entities';
+import useAccountSettings from './useAccountSettings';
+import { convertAmountToNativeDisplay } from '@/helpers/utilities';
+import useCoinListEditOptions from './useCoinListEditOptions';
+
+export function useAssetsBalanceForAddress(address: EthereumAddress) {
```

Comment:
this is essentially the same hook (minus custom network support) as the bx [here](https://github.com/rainbow-me/browser-extension/blob/f82dfd762b6de1b7500ed8dff616a649ba883ef1/src/entries/popup/hooks/useUserAssetsBalance.ts#L20)